ddl/ddl_tiflash_api.go

```diff
@@ -113,10 +121,17 @@ func NewPollTiFlashBackoffContext(MinThreshold, MaxThreshold TiFlashTick, Capaci
 // TiFlashManagementContext is the context for TiFlash Replica Management
 type TiFlashManagementContext struct {
 	TiFlashStores             map[int64]helper.StoreStat
-	HandlePdCounter           uint64
 	UpdateTiFlashStoreCounter uint64
-	UpdateMap                 map[int64]bool
+	PollCounter               uint64
+	ProgressCache             map[int64]string
 	Backoff                   *PollTiFlashBackoffContext
+	AvailableTables           *list.List
```

Comment:
The available is defined as "progress = 1" in most contexts. However the available table here may contain progress < 1 tables. Maybe better to give another name, avoid the usage of "available"?